examples/vg-specs/bar_grouped.vg.json

```diff
@@ -28,9 +28,7 @@
             "url": "data/population.json",
             "format": {
                 "type": "json",
-                "parse": {
-                    "people": "number"
-                }
+                "parse": {}
```

Comment:
@domoritz  not sure if we better output nothing or `{}`.  

(If I understand correctly,  `{}` means don't apply auto-parsing?)